### ISSUE TITLE
Add alt tags to PDP images

### ISSUE THREAD

**Issue by PeterAtger** [scandipwa/scandipwa#5381]
### Is your feature request related to a problem?

All product images have empty alt tags. From an SEO standpoint. All images should have alt tags, in America it's required by law to add alt tags for a11y.

### Describe the solution you'd like to see

By default, all product images should have the product name as an alt tag.

**Comment by noobCode-69** [comment]
Can you please tell how to reproduce the problem?
